build/configs/imxrt1050-evk/README.md

```diff
@@ -0,0 +1,25 @@
+# IMXRT1050-EVK
+
+## Contents
+> [Information](#information)  
+> [Environment Set-up](#environment-set-up)  
+> [How to program a binary](#how-to-program-a-binary)  
+
+## Information
+
+Please refer NXP Semiconductors [homepage](https://www.nxp.com/support/developer-resources/software-development-tools/i.mx-developer-resources/i.mx-rt1050-evaluation-kit:MIMXRT1050-EVK).
+
+## Environment Set-up
+This section covers board-specific environment set-up.  
+Please set TizenRT common environment, [quick start](https://github.com/Samsung/TizenRT#quick-start), first before doing below.
+
+## How to program a binary
```

Comment:
Could you explain the way of flash by manually copying as well?

Reply:
Do you mean the way to copy tynyara.hex made from our build operation into our evk's memory?